### ISSUE TITLE
Ability to cancel just a specific step or subset of steps of an active pipeline run

### ISSUE THREAD

**Issue by gergelylendvai** [dagster-io/dagster#4679]
## Use Case
There is a failing step in a huge pipeline and we would like to let the remaining parts of the pipeline continue. It would be great being able to cancel only the failing part of the graph and re-execute only the failed jobs/steps.

**Comment by sryza** [comment]
@gergelylendvai, when you say "we would like to let the remaining parts of the pipeline continue", is that not what happens already?  I.e. the failing step should cause downstream steps to get stepped, but steps that aren't downstream should still proceed.

**Comment by gergelylendvai** [comment]
Yeah, however in our use case we defined retries for all of our steps and before each run the step is purging the output of the previous run if there was any. So we could just restart the failing part in another pipeline run and let the current one proceed but since we are not able to terminate the retrying step it would just remove the output of the fixed step (in the separate pipeline run).
